sway-lib-std/tests/test_projects/token_ops/mod.rs

```diff
@@ -138,6 +138,70 @@ async fn can_force_transfer() {
     //assert_eq!(balance_result.value, 42);
 }
 
+#[tokio::test]
+async fn can_mint_and_send_to_contract() {
+    let (provider, wallet) = setup_test_provider_and_wallet().await;
+    let (fuelcoin_instance, fuelcoin_id) =
+        get_fuelcoin_instance(provider.clone(), wallet.clone()).await;
+    let balance_id = get_balance_contract_id(provider, wallet).await;
+    let amount = 55u64;
+
+    let asset_id = testfuelcoincontract_mod::ContractId {
+        value: fuelcoin_id.into(),
+    };
+
+    let target = testfuelcoincontract_mod::ContractId {
+        value: balance_id.into(),
+    };
+
+    fuelcoin_instance
+        .mint_and_send_to_contract(amount, target.clone())
+        .set_contracts(&[balance_id])
+        .call()
+        .await
+        .unwrap();
+
+    let result = fuelcoin_instance
+        .get_balance(asset_id, target)
+        .set_contracts(&[balance_id])
+        .call()
+        .await
+        .unwrap();
+
+    assert_eq!(result.value, amount)
+}
+
+async fn can_mint_and_send_to_address() {
+    let (provider, wallet) = setup_test_provider_and_wallet().await;
+    let (fuelcoin_instance, fuelcoin_id) =
+        get_fuelcoin_instance(provider.clone(), wallet.clone()).await;
+    let balance_id = get_balance_contract_id(provider, wallet.clone()).await;
+    let amount = 55u64;
+
+    let asset_id_array: [u8; 32] = fuelcoin_id.into();
+
+    let address = wallet.address();
+    let recipient = testfuelcoincontract_mod::Address {
+        value: address.into(),
+    };
+
+    fuelcoin_instance
+        .mint_and_send_to_address(amount, recipient)
+        .set_contracts(&[balance_id])
```

Comment:
Do we need `set_contracts` here given that we're sending to an `Address`?

Reply:
nope. good catch, copy-pasta error.